src/lib/eZ/RichText/Resources/stylesheets/xhtml5/edit/docbook.xsl

```diff
@@ -256,20 +256,28 @@
   </xsl:template>
 
   <xsl:template name="link.anchor">
+    <xsl:param name="attribute"/>
     <anchor>
       <xsl:attribute name="xml:id">
-        <xsl:value-of select="@id"/>
+        <xsl:value-of select="$attribute"/>
       </xsl:attribute>
     </anchor>
   </xsl:template>
 
-  <xsl:template match="ezxhtml5:a">
+  <xsl:template match="ezxhtml5:a[not(@name=preceding::ezxhtml5:a/@name)]">
```

Comment:
This won't affect links or anchors with id right?

Reply:
Indeed, only the name duplicates will be checked.

Reply:
Side comment: to be fully bulletproof we would need to also check if there are no duplicates when combining both `id` and `name` attributes produced by RTE and external sources mixed together.

Example:
`<a name="test"/><a name="test"/>` ➡️ `<anchor id="test/>test` ✅ 
`<a name="test"/><a id="test"/>` ➡️ `<anchor id="test"><anchor id="test">` :x:

Solution:
```
<xsl:template match="a[not(@id=preceding::a/@id) and 
                       not(@name=preceding::a/@name) and
                       not(@id=preceding::a/@name) and
                       not(@name=preceding::a/@id)]">
```

Reply:
After internal sync with reviewers, we decided to keep the current solution as is.